code/game/objects/structures/headpike.dm

```diff
@@ -3,6 +3,9 @@
 	desc = "When you really want to send a message."
 	icon = 'icons/obj/structures.dmi'
 	icon_state = "headpike"
+	var/obj/item/twohanded/spear/spear = null
+	var/obj/item/bodypart/head/victim = null
+	var/obj/item/twohanded/bonespear/bonespear = null
```

Comment:
just make it obj/spear or obj/twohanded/spear, no reason to have both

Reply:
Fair enough, didn't think about that.